tutorials/generate-events.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: Generating Events
+---
+
+Kite applications work with Big Data. `GenerateEvents.java` generates 1-1.5 million random event records, a small amount of realistic Big Data you can use with Kite examples.
+
+Much of the class is devoted to creating random values. The two methods of interest are `run` and `generateRandomEvent`.
+
+The `run` method performs the following tasks:
+* creates a view of the `hive:events` dataset
+* creates a writer instance
+* spends 36 seconds writing random events
+* closes the writer, which stores the results in the `events` dataset.
+
+While the goal is to create random events, if they're _too_ random there won't be anything to aggregate. The `while` loop simulates a user session with random values for `sessionId`, `userId`, and `ip`. It then generates up to 25 random events for that session.
```

Comment:
> Whatever code we provide as a sample needs to be described, at least in broad strokes.

Agreed, but I want for users to be able to understand what the tutorial accomplishes (add interesting data for later tutorials) and get it done before diving into it because it is mostly an interesting tangent. We should also note that it is optional.

> What would be nice is if we had something interesting that exercised Kite in a practical way

Could you explain what you mean here?